Documentation/gettingstarted/k8s-install-sandbox.rst

```diff
@@ -1,17 +0,0 @@
-.. only:: not (epub or latex or html)
-
-    WARNING: You are looking at unreleased Cilium documentation.
-    Please use the official rendered version released here:
-    https://docs.cilium.io
-
-Creating a Sandbox environment
```

Comment:
I can see how this and other sections made sense in theory, but resulted in users having to do quite a few extra clicks.